backends/cuda/ceed-cuda-operator.c

```diff
@@ -311,7 +311,7 @@ static int CeedOperatorApplyAdd_Cuda(CeedOperator op, CeedVector invec,
   for (CeedInt i = 0; i < numoutputfields; i++) {
     ierr = CeedQFunctionFieldGetEvalMode(qfoutputfields[i], &emode);
     CeedChk(ierr);
-    if (emode == CEED_EVAL_NONE && !impl->eandqdiffer) {
+    if (emode == CEED_EVAL_NONE) {
       // If the E- and Q-Vector layouts are not the same, we will perform
       // an element restriction during the output basis apply, and don't
       // need to set these pointers.  Otherwise, set the output Q-Vector
```

Comment:
First sentence of this comment no longer applies and should be removed.

Reply:
done